internal/project/auto/golang.go

```diff
@@ -42,7 +42,14 @@ func (builder *builder) DetectGolang() (bool, error) {
 
 	builder.meta.CanonicalPath = modfile.ModulePath(contents)
 
-	for _, srcDir := range []string{"src", "internal", "pkg", "cmd", "api"} {
+	for _, srcDir := range []string{
+		"api",         // Kubebuilder's resources
```

Comment:
In COSI/runtime and Theila this folder contains `.proto` files and the generated `.go` output.